<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QMainWindow" name="Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>690</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1220</width>
    <height>690</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2" colspan="2">
     <widget class="QFrame" name="objectDetailsFrame">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>350</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="detailsLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QCheckBox" name="objVisibleCB">
           <property name="text">
            <string>Visible</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="nameLayout">
           <item>
            <widget class="QLabel" name="objNameLbl">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="objDataNameLbl">
             <property name="text">
              <string>Unknown</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="idLayout">
           <item>
            <widget class="QLabel" name="objIdLbl">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="objDataIdLbl">
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="verticesLayout">
           <item>
            <widget class="QLabel" name="objVerticesLbl">
             <property name="text">
              <string>Vertices:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="objDataVerticesLbl">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="edgesLayout">
           <item>
            <widget class="QLabel" name="objEdgesLbl">
             <property name="text">
              <string>Edges</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="objDataEdgesLbl">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="facesLayout">
           <item>
            <widget class="QLabel" name="objFacesLbl">
             <property name="text">
              <string>Faces:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="objDataFacesLbl">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="objectWidthLayout">
           <item>
            <widget class="QLabel" name="objWidthLbl">
             <property name="text">
              <string>Width:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="objDataWidthLbl">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="objectHeightLayout">
           <item>
            <widget class="QLabel" name="objHeightLbl">
             <property name="text">
              <string>Height:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="objDataHeightLbl">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="objectLengthLayout">
           <item>
            <widget class="QLabel" name="objLengthLbl">
             <property name="text">
              <string>Length:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="objDataLengthLbl">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="materialLayout">
           <item>
            <widget class="QLabel" name="objMaterialLbl">
             <property name="text">
              <string>Material:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="objDataMaterialComboBox">
             <item>
              <property name="text">
               <string>Metal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tree</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>219</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QOpenGLWidget" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>602</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QListWidget" name="objsListWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>205</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="objsListLbl">
      <property name="text">
       <string>Objects:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="objectDetailsLbl">
      <property name="text">
       <string>Object Details:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1220</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
